examples/ex-07.c

```diff
@@ -0,0 +1,772 @@
+/*BHEADER**********************************************************************
```

Comment:
Can you add a short description to the examples/README.md file for this example?